docs/diginstroom/sip/2.0/3_core-concepts.md

```diff
@@ -0,0 +1,87 @@
+
+Examples of these three concepts are given in the table below.
+
+| Intellectual Entity | Representation | File |
+|:------------------- | -------------- | ---- |
+| a painting | the stitched high-resolution archive master of the painting | the `.tiff` files that compose the stiched archive master |
+| an episode of a certain TV series | a browse copy of the episode | the `.mov` file and accompagnying `.srt` subtitle file |
+| a newspaper edition of a certain date | an OCR rendition of the newspaper edition | the ALTO `.xml` files containing the newspaper's edition's textual content |
+
+## Content profiles
+
+A generic data model is necessary to ensure a scalable SIP design to current and future use cases.
+However, not every utilization of this model is allowed at ingest in the meemoo archive.
+Depending on the type of content, a specific mapping is required, which is captured in the different [content profiles]({{ site.baseurl }}{% link docs/diginstroom/sip/2.0/profiles/index.md %}).
+A content profile determines
+
+- whether an IE is subdivided further into other IEs (i.e. nesting of IEs);
+- what the representations of any IE are;
+- which files should be included in each representation;
+- what metadata schemas can be used and where.
+
+Content that is delivered to meemoo must therefore always be packaged in a meemoo SIP that adheres to a specific content profile.
+
+{: .important }
+Generally speaking, it is up to the content partner to decide what aspect of its collection is considered as an IE and what profile to use.
+Note that this decision may have an impact on the expressivity of the metadata (eg. the basic profile only supports descriptive metadata in the Dublin Core vocabulary), the dissemination of the SIP's content and, in particular, on how its content is rendered on e.g. meemoo's dissemination platforms.
+Typically, something that has individual descriptive metadata at the source (e.g. an entry in a collection registration or asset management system) and is expected to be distinguishable in search, should be viewed as an IE.
+
+## Example
+
+Consider a digitised newspaper edition with 10 pages.
+Each page is digitised separately as a TIFF file and a JPEG file, resulting in 20 digital reproductions.
+Finally, OCR is applied to each TIFF file, resulting in yet another 10 ALTO XML files (containing the textual representation of each newspaper page).
+Using the concepts defined above, we view the newspaper edition as a whole as the IE.
+
+We have one IE (i.e. the newspaper edition) and we can discern between three representations of that IE: a high-resolution representation containing the TIFF files for e.g. reproduction, a low-resolution representation containing the JPEG files for e.g. browse copies on the web and a textual representation containing the ALTO XML files for e.g. searching through the actual textual content of the newspaper pages.
```

Comment:
We might want to update this section, since we model it differently in the meantime.